doc/dev/BUILD.md

```diff
@@ -10,6 +10,7 @@ please take a look at our [getting started guide](GETTING_STARTED.md).
 * [VTK](https://vtk.org) >= 9.0.0 (9.2.6 recommended).
 * A C++17 compiler.
 * A CMake-compatible build system (Visual Studio, XCode, Ninja, Make, etc.).
+* Required for running tests, [git LFS](https://git-lfs.com/)
```

Comment:
I dont think this is the right location for this, as this is really specific to people cloning the repository using git instead of downloading the tarbal. Not needed here imo.

Reply:
but you could mention it next to the BUILD_TESTING option ?

Reply:
So don't mention at all, then? Well, given links/mentions of git LFS it should be easy to figure it out, I guess